src/NodaTime/DateTimeZoneProviders.cs

```diff
@@ -81,5 +84,36 @@ public static IDateTimeZoneProvider Serialization
                 }
             }
         }
+
+        /// <summary>
+        /// Gets the <see cref="IDateTimeZoneProvider"/> to use to interpret a time zone ID read as part of
+        /// conversion using the default <see cref="System.ComponentModel.TypeConverter"/> for <see cref="ZonedDateTime"/>.
+        /// Note that if a value other than <see cref="DateTimeZoneProviders.Tzdb"/> is required, it should be set on
+        /// application startup, before any type converters are used. Type converters are cached internally by the framework,
+        /// so changes to this property after the first type converter for <see cref="ZonedDateTime"/> will not generally
```

Comment:
Missing a word. Also I wonder if you'd be better off being less-specific here, and just saying "changes to this property after any type converters have been created will not generally take effect" or similar?

Reply:
While I could do that, I think it's more likely to be useful this way, because we know that type converters *are* currently cached (and that's unlikely to change, although it's possible). Type converters are often fetched implicitly, so it's not necessarily clear to the user when the type converter would be created - explicitly calling out the caching should clarify things.